src/Compilers/CSharp/Test/Symbol/Symbols/InterfaceOverriddenOrHiddenMembersTests.cs

```diff
@@ -819,5 +819,361 @@ public interface Test { }
 }";
             CreateCompilationWithMscorlib(source).VerifyDiagnostics();
         }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.ReadonlyReferences)]
+        public void HidingMethodWithRefReadOnlyParameterWillProduceAWarning()
```

Comment:
Consider dropping `Will` and `WillNotProduceAWarning` suffixes, for consistency with other tests. (The tests themselves indicate whether a warning is produced.)